### PR TITLE
Develop v01

### PR DESCRIPTION
**Pull request by Kelvin-Ben**
## Full working app
- This app shows updates on covid 19 worldwide.
- Through a selection of different countries, one can get information about the virus spread

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,13 @@
 - HTML
 - CSS
 - JavaScript
+- Tailwind
 
 ## Live Demo :arrow_forward:
```
Comment (mwanawabangona) [inline README.md]: - [ ] `required`
- [ ] Please add a live demo as the requirements state. 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @Kelvin-Ben ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- When presenting you project on video you talk about what you like or loved and the challenges you went through.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kelvin-Ben) [reply README.md]: @mwanawabangona I am working on a feature branch that will have to merge for the default branch to be deployable

Comment (danielufeli) [reply README.md]: @Kelvin-Ben you can deploy the feature branch am trying to reach you on slack

Comment (Kelvin-Ben) [reply README.md]: okay

Comment (Kelvin-Ben) [reply README.md]: @danielufeli can you guide me please

**Review by danielufeli — APPROVED** [review]
Hi @Kelvin-Ben,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

ChopApproveCopApproveGIF

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Clean Code✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._ate_comments_counter`

**Pull request merged**